micrometer-registry-stackdriver/deployment/src/main/java/io/quarkiverse/micrometer/registry/stackdriver/deployment/StackdriverRegistryProcessor.java

```diff
@@ -36,6 +38,13 @@ public boolean getAsBoolean() {
         }
     }
 
+    @BuildStep(onlyIf = { StackdriverEnabled.class, NativeBuild.class })
```

Comment:
Needed to fix Stackdriver build in native mode... it broke with the bump to Quarkus 2.10.0.Final...